azext_iot/digitaltwins/providers/twin.py

```diff
@@ -92,7 +92,9 @@ def update(self, twin_id, json_patch, etag=None):
         elif isinstance(json_patch, list):
             json_patch_collection.extend(json_patch)
         else:
-            raise CLIError(f"--json-patch content must be an object or array. Actual type was: {type(json_patch).__name__}")
+            raise InvalidArgumentValueError(
+                f"--json-patch content must be an object or array. Actual type was: {type(json_patch).__name__}"
```

Comment:
does type(json_patch) not work?

Reply:
This wasn't changed in the PR. type(json_patch) gives a string representation of the object class (usually object.__class__) where-as .__name__ gives just the class name, which is useful when you want a specific msg format.